src/index.ts

```diff
@@ -15,19 +15,19 @@ import {
 
 
   (async function () {
-    const wh = new Wormhole("Testnet", [solana.Platform, evm.Platform], {
+    const wh = new Wormhole("Testnet", [evm.Platform], {
       // optional way to use private RPCs, especially recommended for mainnet 
       "chains": {
-        "Monad": {
-          "rpc": "http://127.0.0.1:8546"
+        "Sepolia": {
+          "rpc": "https://eth-sepolia.g.alchemy.com/v2/g78WWQ0EDYu4KqOX3KsrxF2VKk6LOHHd"
```

Comment:
Can we extract URL RPC with a private key to .env?
I know that for Mezo doesn't have a private key, but for consistency I'd do the same.